### PR TITLE
Valid section renderer

### PR DESCRIPTION
**Pull request by siva-kannan3**
#### Changes

if content of a section is empty, removed the section from the layout.

Fixes # (issue)

#### Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

#### Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings

### CODE REVIEW

File: src/helpers/common/components/ValidSectionRenderer.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import { Fragment, ReactNode, useMemo } from 'react';
+import { IExperienceItem } from 'src/stores/experience.interface';
+import { IEducationItem } from 'src/stores/education.interface';
+import { IAwardItem } from 'src/stores/awards.interface';
+import { IVolunteeringItem } from 'src/stores/volunteering.interface';
+import { ISkillItem } from 'src/stores/skill.interface';
+
+export const ValidSectionRenderer = ({
```
Comment (gopal1996) [inline src/helpers/common/components/ValidSectionRenderer.tsx:8]:
```suggestion
export const SectionValidator = ({
```
is it conveying @siva-kannan3 ?

Comment (siva-kannan3) [reply src/helpers/common/components/ValidSectionRenderer.tsx:8]: yeah

File: src/templates/modern/MordernTemplate.tsx [hunk 2]
```diff
@@ -7,6 +7,7 @@ import { WorkSection } from './components/Work';
 import { AwardSection } from './components/Awards';
 import { useContext } from 'react';
 import { StateContext } from 'src/modules/builder/resume/ResumeLayout';
+import { ValidSectionRenderer } from 'src/helpers/common/components/ValidSectionRenderer';
```
Comment (gopal1996) [inline src/templates/modern/MordernTemplate.tsx:10]: can you update in import side also

Comment (siva-kannan3) [reply src/templates/modern/MordernTemplate.tsx]: done bro

File: src/templates/modern/MordernTemplate.tsx [hunk 3]
```diff
@@ -7,6 +7,7 @@ import { WorkSection } from './components/Work';
 import { AwardSection } from './components/Awards';
 import { useContext } from 'react';
 import { StateContext } from 'src/modules/builder/resume/ResumeLayout';
+import { SectionValidator } from 'src/helpers/common/components/SectionValidator';
```
Comment (gopal1996) [inline src/templates/modern/MordernTemplate.tsx:10]: filename not changed @siva-kannan3

**Review by gopal1996 — APPROVED** [review]

**Pull request merged**